src/Windows/Avalonia.Win32/Win32GlManager.cs

```diff
@@ -20,13 +22,11 @@ public static void Initialize()
                     return wgl;
                 }
 
-                if (opts?.AllowEglInitialization == true ||
-                    ((!opts?.AllowEglInitialization.HasValue ?? false) &&
-                     Win32Platform.WindowsVersion > new Version(6, 1)))
+                if (opts?.AllowEglInitialization ?? Win32Platform.WindowsVersion > Windows7)
                 {
                     var egl = EglPlatformOpenGlInterface.TryCreate(() => new AngleWin32EglDisplay());
 
-                    if (egl is { } &&
+                    if (egl != null &&
```

Comment:
C#9 introduced the `is not` operator. So it seems that this unnecessary change will be in this pr :)

Reply:
This seemed like fancyness for fancess sake. It's a not-null check....

Reply:
However, this is how we can write https://dotnetfiddle.net/0AgVfr

Using this hack to avoid writing  `!(string.Empty is null)`